auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -367,8 +368,8 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     }
     // if this is an IA or registration then show legal type
     if (this.isTemporaryBusiness(business)) {
-      const legalType: unknown = business.corpType.legalType
-      return GetCorpFullDescription(legalType as CorpTypeCd)
+      const legalType: unknown = business.corpSubType?.code
+      return GetCorpFullDescription(legalType as CorpTypeCd) // may return ''
```

Comment:
Temporary business = IA or Registration

In this case, look at the new object to get the legal type.